src/Indicator.vala

```diff
@@ -214,6 +218,24 @@ public class Power.Indicator : Wingpanel.Indicator {
 
         return false;
     }
+
+    private bool battery_charged_notification () {
+      if (is_in_session) {
+          try {
+              var notification = new Notify.Notification (
+                  _("%s fully charged").printf (display_device.device_type.get_name ()),
+                  _("Unplugging the device now can extend battery health"), "battery-full-charged");
```

Comment:
Sorry to be picky but "extend" means "to make longer" (and "health" is a quality without length) so it should either be "extend battery life" or "improve battery health"

Reply:
Done 7e7b306!